internal/lsp/completions/providers/policy_test.go

```diff
@@ -0,0 +1,50 @@
+package providers
+
+import (
+	"testing"
+
+	"github.com/open-policy-agent/opa/ast"
+
+	"github.com/styrainc/regal/internal/lsp/cache"
+	"github.com/styrainc/regal/internal/lsp/types"
+)
+
+func TestPolicyProvider(t *testing.T) {
+	t.Parallel()
+
+	locals := NewPolicy()
```

Comment:
I wonder if there's a way here to make it so that only one of the Rego evaluators is run, so that when we have more, testing them individually is easier.

Reply:
The Rego providers are "all" (well, in the future anyway) thoroughly tested in Rego. What we do here should be closer to an integration test... but you're still right — it'd be good to be able to have at least a very basic such test for each provider. Let's look into that as we add more.